Documentation/ViewHelperReference.md

```diff
@@ -279,3 +279,51 @@ Usage with EXT:form:
 	</my:molecule.fieldLabel>
 </fc:form.translatedValidationResults>
 ```
+
+## Mapping
+
+### Variable.Map ViewHelper
+
+Components should not contain dependencies on how they receive data.
+These can be DataProviders, mask  elements or news items, for example.
+This ViewHelper takes an array of data items (Objects/Arrays) and make a new array with the new keys configured in fieldsMapping and optionally the existing fields specified in the keepFields.
+
+#### Arguments
+
+* `subject`: Source array of data items. If not in arguments then taken from inline construction.
+* `fieldsMapping`: Array of mapping keys. (see examples)
+    * `key`: New key
+    * `value`: Name or dot-separated path to field in source array
+* `keepFields`: Array or comma separated list of fields to keep in array. Is optional.
```

Comment:
Use ` (optional)` after argument name, just like in ViewHelpers documented above.